quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliConfigEncryptIT.java

```diff
@@ -170,32 +170,17 @@ public void testWrongSecretKeyHandler() {
     public void testEncryptCommandHelp() {
         encryptBuilder
                 .printOutHelp()
-                .assertCommandOutputContains("""
-                        Encrypt a Secret value using the AES/GCM/NoPadding algorithm as a default
-                        """)
-                .assertCommandOutputContains("""
-                        encryption key is generated unless a specific key is set with the --key option
-                        """)
-                .assertCommandOutputContains("""
-                        Usage: quarkus config encrypt [-eh] [--verbose] [-f=<encryptionKeyFormat>]
-                        """)
+                .assertCommandOutputContains("Encrypt a Secret value using the AES/GCM/NoPadding algorithm as a default")
+                .assertCommandOutputContains("encryption key is generated unless a specific key is set with the --key option")
+                .assertCommandOutputContains("Usage: quarkus config encrypt [-eh] [--verbose] [-f=<encryptionKeyFormat>]")
                 .assertCommandOutputContains("[-k=<encryptionKey>] SECRET")
-                .assertCommandOutputContains("""
-                        The secret value to encrypt
-                        """)
                 .assertCommandOutputContains("-f, --format=<encryptionKeyFormat>")
-                .assertCommandOutputContains("""
-                        The encryption key format (base64 / plain)
-                        """)
-                .assertCommandOutputContains("""
-                        Print more context on errors and exceptions
-                        """)
-                .assertCommandOutputContains("""
-                        Display this help message
-                        """)
+                .assertCommandOutputContains("The Encryption Key Format (base64 / plain)")
+                .assertCommandOutputContains("Print more context on errors and exceptions")
+                .assertCommandOutputContains("Display this help message")
                 .assertCommandOutputContains("Verbose mode")
                 .assertCommandOutputContains("-k, --key=<encryptionKey>")
-                .assertCommandOutputContains("The encryption Key");
+                .assertCommandOutputContains("The Encryption Key");
```

Comment:
Afaik I don't think this should change as it was changed in https://github.com/quarkus-qe/quarkus-test-suite/pull/2788 to reflect upstream changes. The change wasn't backported to 3.27 branch.

I don't mind that much changing this file from using multiline string to single line, but maybe if no change is needed I would leave this file untouched.

Reply:
I executed it with `mvn -fae -V -B clean verify -Doc.reruns=0 -Dreruns=0 -Proot-modules -f quarkus-cli -Dit.test="QuarkusCliConfigEncryptIT"`

Reply:
Fixed, verified with quarkus cli 3.10.1 (from sdkman)